### DISCUSSION TITLE
Add a license

### DISCUSSION THREAD

**Discussion by h7x4** [ros-industrial/packet-simplemessage#30] [General]
It would be nice if you could distribute this project under a license, so it's easier to redistribute (or know not to redistribute) downstream.

**Comment by gavanderhoorn** [comment]
Could you clarify what you feel is missing exactly?

The dissector essentially consists of a single file, and the license is stated at the top of the file:

https://github.com/ros-industrial/packet-simplemessage/blob/b3a655b227a76e29912b8148020908202fc9d4cb/packet-simplemessage.lua#L2-L18

like many other Wireshark dissectors, it's licensed under the GPLv2.

**Comment by h7x4** [comment]
Hello!

I didn't catch that first time I went looking. I expected a `LICENSE` file to exist, but this works as well. Sorry for taking your time!

**Reply by gavanderhoorn** [reply]
No problem.

I noticed you are [packaging this dissector for Nix](https://github.com/h7x4/nixpkgs/commit/6aa3b4c2320c5c25895da85bdbca2da446194e68). Is this a personal interest, or are there actually people interested in `package-simplemessage`?

**Reply by h7x4** [reply]
Correct, that was the original reason I asked. This is more of a personal interest for now. I'm planning to try setting up plugin infrastructure for nixpkgs, to encourage both use and development of stuff like dissectors, extcap plugins, menu plugins, etc. There doesn't seem to be a lot of quality plugins out there at the moment (the wireshark team seems to merge stuff once it's good/big enough?), so I'm just packing everything I can find with decent quality to jumpstart the project. They will act as reference when packing more in the future.
